src/contracts/pods/EigenPodStorage.sol

```diff
@@ -11,7 +11,7 @@ abstract contract EigenPodStorage is IEigenPod {
     uint64 internal __deprecated_mostRecentWithdrawalTimestamp;
 
     /// @notice the amount of execution layer ETH in this contract that is staked in EigenLayer (i.e. withdrawn from the Beacon Chain but not from EigenLayer),
-    uint64 public withdrawableRestakedExecutionLayerGwei;
+    uint64 internal restakedExecutionLayerGwei;
```

Comment:
yes! but this isn't a breaking change because i added a `public` function with the same name as the state variable. 

So the UI doesn't need to know about this - it'll function exactly the same.

Reply:
Ah, did not see that one. Perfect